### PR TITLE
Various 202301 release hotfixes

### PR DESCRIPTION
**Pull request by mereolog**
Fixes (among other things): #37

### CODE REVIEW

**Comment by wsobel** [comment]
Why is `master` in the versionIRI?

**Comment by wsobel** [comment]
As in `https://spec.industrialontologies.org/ontology/` **`master`** `/202301/MetadataIOF/`

**Comment by mereolog** [comment]
> As in `https://spec.industrialontologies.org/ontology/` **`master`** `/202301/MetadataIOF/`

This is how the current publishing process works. Since you can tag any branch, we need to know which branch a given tag belongs to.

**Comment by wsobel** [comment]
Seemed like a conflation of a VC artifact. I'll approve, maybe we can discuss later.

**Review by wsobel — APPROVED** [review]

File: core/Metadatacore.rdf [hunk 1]
```diff
@@ -32,8 +32,8 @@
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/"/>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/meta/MappingAnnotationVocabularyToCommons/"/>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"/>
-		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/20230120/core/Metadatacore/"/>
-		<iof-av:copyright>Copyright (c) 2023 Open Applications Group</iof-av:copyright>
+		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/master/202301/core/Metadatacore/"/>
```
Comment (kbserm) [inline core/Metadatacore.rdf:35]: Why is there the word 'master' in the IRI?

Comment (mereolog) [reply core/Metadatacore.rdf:35]: > Why is there the word 'master' in the IRI?

See https://github.com/iofoundry/ontology/pull/40#issuecomment-1412038889 above and @wsobel reply.

**Comment by chris-will** [comment]
Changes to the changenotes for Annotation Vocabulary file are missing. Please incorporate latest updates I just emailed you.

**Comment by mereolog** [comment]
> Changes to the changenotes for Annotation Vocabulary file are missing. Please incorporate latest updates I just emailed you.

Done, I hope, in [fb3c39f](https://github.com/iofoundry/ontology/pull/40/commits/fb3c39f0581f85c4c959222747892ee8d29938c7).

@chris-will could you verify?

**Review by chris-will — APPROVED** [review]

**Comment by chris-will** [comment]
Pawel, changenotes look good. I had to delete some of the old ones directly in this pull request. They were still lingering in there after you inserted the newly formatted ones.

**Review by chris-will — COMMENTED** [review]
I also added the provisional statements to terms 'event' and 'material state'

**Review by ekwallace — APPROVED** [review]
Does "master" show up in the path of the versionIRI in the published version or the version in the GitHub repo after merging the branch?  Sounds like something else that we need to talk about in the version metadata discussion in the Arch TG meeting on 2 Feb 23.

**Comment by mereolog** [comment]
> Does "master" show up in the path of the versionIRI in the published version or the version in the GitHub repo after merging the branch? Sounds like something else that we need to talk about in the version metadata discussion in the Arch TG meeting on 2 Feb 23.

In both.
For the published version it is automatic.
For the Github version it is manual, so it can be removed.

File: core/commonstocoremapping/MappingCommonsToIOF.rdf [hunk 2]
```diff
@@ -47,9 +47,9 @@
 		<owl:imports rdf:resource="https://www.omg.org/spec/Commons/Collections/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/Commons/Designators/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/Commons/Identifiers/"/>
-		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/202202/core/MappingCommonsToIOFCore/"/>
-		<owl:versionInfo rdf:datatype="&xsd;integer">202202</owl:versionInfo>
-		<iof-av:copyright>Copyright (c) 2022 Open Applications Group</iof-av:copyright>
+		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/master/202301/core/MappingCommonsToIOFCore/"/>
```
Comment (przemekgradzki) [inline core/commonstocoremapping/MappingCommonsToIOF.rdf:50]:
```suggestion
		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/master/202301/core/commonstocoremapping/MappingCommonsToIOF/"/>
```

**Comment by przemekgradzki** [comment]
https://github.com/iofoundry/ontology/blob/bf71e6718461365d0856f47722e182a2867b087f/core/Metadatacore.rdf#L32

should be:
```
 <owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOF/"/> 
```

File: core/Metadatacore.rdf [hunk 3]
```diff
@@ -29,11 +29,11 @@
 		<dct:issued rdf:datatype="&xsd;dateTime">2023-01-20T00:00:00</dct:issued>
 		<dct:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dct:license>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/Core/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/"/>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOFCore/"/>
```
Comment (przemekgradzki) [inline core/Metadatacore.rdf:32]:
```suggestion
		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOF/"/>
```

see [comment](https://github.com/iofoundry/ontology/pull/40#issuecomment-1413813360)

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

**Review by przemekgradzki — APPROVED** [review]

**Pull request merged**